certbot-auto

```diff
@@ -131,7 +131,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO+x}" ]; then
```

Comment:
> ${variable+x} is always going to evaluate to 'x'

`${LE_AUTO_SUDO+x}` evaluates to `x` if and only if `$LE_AUTO_SUDO` is set [1].

That seems to be the intended behavior of this line; if `$LE_AUTO_SUDO` is set, we use its value to set `$SUDO`, otherwise we determine the appropriate value for `$SUDO`  based on the presence of `sudo`.

1 - For more information, see: www.tldp.org/LDP/abs/html/parameter-substitution.html#PARAMALTV